Source/ARTClientOptions.m

```diff
@@ -194,4 +201,16 @@ + (BOOL)getDefaultIdempotentRestPublishingForVersion:(NSString *)version {
     }
 }
 
+- (BOOL)isProductionEnvironment {
+    return [self.environment.lowercaseString isEqualToString:ARTDefaultProduction];
```

Comment:
This should be lowercased on the left and right sides to be 100% sure that comparing will work properly.

Reply:
changed here d6e21a005f29bb7e0532b41903e1832234267880